examples/python/prerequisites.md

```diff
@@ -1,10 +1,18 @@
 # Prerequisites for running the Python examples
-To run the Python SSE plugin examples, you need __Python__ version 3.4 or higher. You can find links for installing Anaconda (a Python distribution including common libraries and packages) on the [Anaconda webpage](https://www.continuum.io/downloads). For Python without any libraries, see the [Python webpage](https://www.python.org/downloads/).
+To run the Python SSE plugin examples, you need __Python__ version 3.4 or higher along with a few _python libraries_.
+
+Anaconda is a Python distribution pre-bundled with multiple extra libraries. An installer can be downloaded from the [Anaconda webpage](https://www.continuum.io/downloads). For a leaner installation the default installer can be found on the webpage of the [Python Software Foundation](https://www.python.org/downloads/).
 
 The following _python libraries_ are needed for the specified SSE plugins.
```

Comment:
End the sentence with colon instead of full stop
"The following _python libraries_ are needed for the specified SSE plugins:"